scripts/metrics/render_json.py

```diff
@@ -162,7 +170,15 @@ def create_obj(
 visii.set_camera_entity(camera)
 
 visii.set_dome_light_intensity(1)
-visii.set_dome_light_color(visii.vec3(1,1,1),0)
+
+try:
+    visii.set_dome_light_color(visii.vec3(1, 1, 1), 0)
+except TypeError:
+    # Support for alpha transparent backgrounds was added in nvisii ef1880aa,
+    # but as of 2022-11-03, the latest released version (1.1) does not include
+    # that change yet.
+    print("WARNING! Your version of NVISII does not support alpha transparent backgrounds yet; --contour will not work properly.")
```

Comment:
Good catch I like it. :P We are working on making the alpha changes on pip, but things get in the way.